### ISSUE TITLE
Can't get CSS to render locally

### ISSUE THREAD

**Issue by phrequencyviii-zz** [Wyamio/Wyam.Web#11]
So I still can't get my Output site to display any CSS stylings when I run the --preview switch.

I'm building off of your blog source, and when I try and preview that untouched, same problem. I've tried Edge, IE, and Chrome. It appears to be finding the CSS files just fine according to F12. I downloaded the Clean Blog theme from Start Bootstrap and that renders fine locally. I'm so confused what I'm somehow doing wrong. Was thinking about pushing the site to GH Pages with AppVeyer like described and see if that shows up fine but I'd like it to work locally...

**Comment by daveaglick** [comment]
Any chance you could send over a full archive of what you're working with? Just zip everything up and attach it to this issue.

**Comment by phrequencyviii-zz** [comment]
So I was originally doing this on a Surface 3, but I returned it and I just set up a temporary Windows 10 craptop to use for now. Here is what I did:
- Downloaded wyam, unzipped to it's own folder with 7zip
- Downloaded your blogs source, unzipped to it's own folder with 7zip
- Opened a CMD prompt in the Wyam folder, and put in **wyam.exe C:\Users\phreq\Downloads\daveaglick-master\daveaglick-master --preview**
- Went to the Output folder and opened index.html in Edge

The page hangs for awhile, then displays with no CSS. It looks like everything else works, though. I don't know if I'm just making some rookie mistake somewhere or what, but this is the only problem I'm having, go figure lol. Thanks for your time.

I just realized I opened this issue on the Wyam website repo and not the Wyam reop. D'oh. Trying to balance too many tabs on a 10" screen I guess. :)
